functional/churn/playlist.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/resources/playlist.xsd">
+	<test>
+		<testCaseName>churn</testCaseName>
+<!--            export DURATION=18000 ; \-->
+		<command> 
+            export OTOOL_JDK_VERSION=$(JDK_VERSION) ; \
+            export JREJDK="jdk" ; \
+            export OTOOL_garbageCollector=ALL ; \
+            export DURATION=24 ; \
+            export TMPRESULTS=$(Q)$(REPORTDIR)$(D)report$(Q); \
+            cd $TMPRESULTS ; \
+              bash $(TEST_ROOT)$(D)functional$(D)churn$(D)churn$(D)run.sh; \
+            $(TEST_STATUS)
+		</command>
+		<levels>
+			<level>dev</level>
+		</levels>
+		<platformRequirementsList>
+			<platformRequirements>os.linux</platformRequirements>
+			<platformRequirements>os.win</platformRequirements>
+			<platformRequirements>os.osx</platformRequirements>
+		</platformRequirementsList>
+		<groups>
+			<group>functional</group>
+		</groups>
+	</test>
+	<test>
+		<testCaseName>churn-nocoops</testCaseName>
```

Comment:
We already have a mechanism to automatically run all test targets twice, once with compressedrefs JVM option, and a second time as noncompressedrefs JVM option.  

I have to look closer at what NOCOMP does in the churn code, is it used for other decision-making within the test code to best advise.